sandbox-templates/mdx-evaulate-node/package.json

```diff
@@ -8,9 +8,9 @@
     "start": "nodemon index.js"
   },
   "dependencies": {
-    "@mdx-js/mdx": "2.1.1",
-    "nodemon": "2.0.15",
-    "react": "17.0.2",
-    "react-dom": "17.0.2"
+    "@mdx-js/mdx": "^2.0.0",
+    "nodemon": "^2.0.0",
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0"
```

Comment:
Currently looking into this causing
```
[mdx-evaulate-node/package.json](https://github.com/mdx-js/.github/pull/48/files#diff-4578cca0b89275f94742b41d31c70ab62c7168ae83f775b5a57f6f5287016bed)
```

Reply:
fixed, they remapped their exports without extensions https://github.com/facebook/react/blob/8a9f82ed58c2fa76583a041fa34ad80f5f94a3d1/packages/react/package.json#L28